File: JJB/1125/피자_나눠_먹기_(2).cpp
#include <string>
#include <vector>

using namespace std;

int gcd(int a, int b)
{
    int c;
    while(b)
    {
        c = b;
        b = a % b;
        a = c;
    }
    return a;
}

int lcm(int a, int b)
{
    return a * b / gcd(a, b);
}

int solution(int n) 
{
    int answer = 0;
    
    answer = lcm(n, 6) / 6;
    
    return answer;
}

File: JJB/231125/피자_나눠_먹기_(2).py
def gcd(a, b):
    while(b):
        a, b = b, a%b
    return a

def lcm(a, b):
    return a * b / gcd(a, b)

def solution(n):
    answer = 0
    
    answer = lcm(n, 6) / 6
    
    return answer